server/test/conf/MessagesTest.java

```diff
@@ -92,6 +92,32 @@ public void messages_valuesEqualMessagesEnUsValues() throws Exception {
         .containsAllEntriesOf(entriesInEnUsLanguageFile);
   }
 
```

Comment:
Notably, below we check the primary language file many times due to parameterization. I realize we could extract this into its own test, but I don't see the need. Feel free to push back.

Reply:
Mmm I don't feel terribly strongly but since you suggested it, having it be its own test might be slightly cleaner.

Also it would be good to test for smart/curly single quotes as well, and it might be more reliable to search for the specific unicode codes:

Character | Unicode
-- | --
“ | \u201C
” | \u201D
‘ | \u2018
’ | \u2019

Reply:
Refactored out the primary language check into two methods.

About the unicocde codes, that's an excellent idea, but because of the way the `messages` files are being read in I don't seem to be able to check for them. They get converted somehow into the weird `â` character (I guess in a similar way as how `''` is changed to `'`.

That being said, all four cases are caught by this weird character, so I guess it's working.